docs/built-in processors.rst

```diff
@@ -0,0 +1,155 @@
+.. _built-in processors:
```

Comment:
could you please avoid whitespaces in the file names? processors.rst?

Reply:
processors.rst already exists, so going with built-in-processors